### DISCUSSION TITLE
关于Caddy中forwardproxy插件的问题

### DISCUSSION THREAD

**Discussion by Bryan2333** [lxhao61/integrated-examples#194] [General]
我看一下github action文件，forwardproxy插件是使用了Sagernet分支，而没有使用Naiveproxy的分支。
```
--with github.com/caddyserver/forwardproxy@caddy2=github.com/sagernet/forwardproxy@latest \
```
请问一下，这有什么原因吗？这个分支已经两年没有更新了，和上游有不少[差别](https://github.com/SagerNet/forwardproxy/compare/naive...klzgrad%3Aforwardproxy%3Anaive)

**Comment by lxhao61** [comment]
Sagernet 分支支持 UoT。
